auth/src/test/java/com/firebase/ui/auth/ui/ChooseAccountActivityTest.java

```diff
@@ -1,150 +0,0 @@
-        ChooseAccountActivity chooseAccountActivity =
-                Robolectric.buildActivity(ChooseAccountActivity.class)
-                        .withIntent(startIntent)
-                        .create()
-                        .get();
-
-        when(mCredentialsAPI.getEmailFromCredential()).thenReturn(TestConstants.EMAIL);
-        when(mCredentialsAPI.getPasswordFromCredential()).thenReturn(TestConstants.PASSWORD);
-        when(mCredentialsAPI.getAccountTypeFromCredential()).thenReturn(
-                EmailAuthProvider.PROVIDER_ID);
-
-        when(mActivityHelper.getFirebaseAuth()).thenReturn(mFirebaseAuth);
-        when(mActivityHelper.getFlowParams()).thenReturn(
-                (FlowParameters) startIntent.getParcelableExtra(ExtraConstants.EXTRA_FLOW_PARAMS));
-
-        when(mFirebaseAuth.signInWithEmailAndPassword(
-                TestConstants.EMAIL,
-                TestConstants.PASSWORD))
-                .thenReturn(
-                        new AutoCompleteTask<AuthResult>(
-                                new FakeAuthResult(mock(FirebaseUser.class)), true, null));
-
-        chooseAccountActivity.onCredentialsApiConnected(mCredentialsAPI, mActivityHelper);
-
-        verify(mFirebaseAuth).signInWithEmailAndPassword(
-                TestConstants.EMAIL,
-                TestConstants.PASSWORD);
-
-        assertTrue(Shadows.shadowOf(chooseAccountActivity).isFinishing());
-    }
-
-    @Test
-    public void testAutoSignInWithSavedIdp_redirectsToIdpSignIn() {
-        Intent startIntent = createStartIntent();
-        ChooseAccountActivity chooseAccountActivity =
-                Robolectric.buildActivity(ChooseAccountActivity.class)
-                        .withIntent(startIntent)
-                        .create()
-                        .get();
-
-        when(mCredentialsAPI.getEmailFromCredential()).thenReturn(TestConstants.EMAIL);
-        when(mCredentialsAPI.getPasswordFromCredential()).thenReturn(null);
-        when(mCredentialsAPI.getAccountTypeFromCredential()).thenReturn(
-                IdentityProviders.GOOGLE);
-        when(mActivityHelper.getFlowParams()).thenReturn(
-                (FlowParameters) startIntent.getParcelableExtra(ExtraConstants.EXTRA_FLOW_PARAMS));
-
-        chooseAccountActivity.onCredentialsApiConnected(mCredentialsAPI, mActivityHelper);
-
-        ShadowActivity.IntentForResult nextIntent = Shadows
-                .shadowOf(chooseAccountActivity)
-                .getNextStartedActivityForResult();
-
-        assertEquals(
-                IdpSignInContainerActivity.class.getName(),
-                nextIntent.intent.getComponent().getClassName()
-        );
-        assertEquals(
-                TestConstants.EMAIL,
-                nextIntent.intent.getExtras().getString(ExtraConstants.EXTRA_EMAIL));
-        assertEquals(
-                GoogleAuthProvider.PROVIDER_ID,
-                nextIntent.intent.getExtras().getString(ExtraConstants.EXTRA_PROVIDER)
-        );
```

Comment:
I removed this test that was making sure the email/password survived the intent.